input/udp.md

```diff
@@ -173,5 +173,10 @@ If `in_udp` gets lots of packets within 1 sec, some packets are dropped. For exa
 
 This means that `in_udp` with one process cannot handle such traffic loads. Try [multi workers](../deployment/multi-process-workers.md).
 
+### Issues with newlines
+
+In edge situations please try to set `remove_newline` to `false` so that input will be properly processed, especially when sendig multiline json messages.
```

Comment:
Adding the note is good, thanks for the notifying it.
But I think the expression seems too abstracted, so I'll add more concrete explanation.

Reply:
I added some concrete examples in a639260. 

I hope it suffices to keep out further confusion among users.